src/analysis/retail/druid/restoration/modules/features/Checklist/Component.tsx

```diff
@@ -11,7 +10,9 @@ import GenericCastEfficiencyRequirement from 'parser/shared/modules/features/Che
 import Requirement from 'parser/shared/modules/features/Checklist/Requirement';
 import Rule from 'parser/shared/modules/features/Checklist/Rule';
 import PropTypes from 'prop-types';
+import { TALENTS_DRUID } from 'common/TALENTS';
 
+// TODO remove in favor of Guide (once UI for this figured out)
```

Comment:
fwiw i went ahead and nuked the brew checklist. don't need to for resto (yet) but it made my life a lot easier